src/pages/player/player.jsx

```diff
@@ -0,0 +1,104 @@
+var React = require('react');
+var {parseJSON} = require('../../../node_modules/webmaker-core/src/lib/jsonUtils');
```

Comment:
I wonder if there is anyway we can expose these components/lib nicely here without having to access them using relative path which is a bit gross IMO?

Can we maybe optimize this later to have `index.js` file where we can require that will expose each of these or something? what do you think?